synapse/tests/test_cortex.py

```diff
@@ -7233,3 +7233,37 @@ async def test_cortex_storage_v1(self):
             msgs = await core.stormlist('macro.exec woot')
             self.stormHasNoWarnErr(msgs)
             self.stormIsInPrint('hi there', msgs)
+
+    async def test_cortex_depr_props_warning(self):
+
+        pattern = 'is unlocked and unused. Recommend locking.'
+
+        with self.getTestDir() as dirn:
+            with self.getLoggerStream('synapse.cortex', pattern) as stream:
+                # Do something that triggers a log message
+                async with self.getTestCore(dirn=dirn) as core:
+
+                    # Create a lang:idiom so it doesn't generate a warning
+                    await core.callStorm('[lang:idiom=foobar]')
```

Comment:
instead of relying on lang:idiom would we could use the `synapse.tests.utils.DeprModule` which loads a model with deprecated information in it. That would survive a model flag day which could remove that `lang:idiom` model.

```
        conf = {'modules': [('synapse.tests.utils.DeprModule', {})]}
        async with self.getTestCore(conf=conf) as core:
            ...
```

Reply:
I was wondering about that. Thanks for the tip on DeprModule.

I updated `test:deprprop` with a secondary property of `ext` which is also deprecated so I could test both using and locking nodes at the same time.